packages/javascript/bh-shared-ui/src/components/ExploreTable/useExploreTableRowsAndColumns.tsx

```diff
@@ -170,10 +186,14 @@ const useExploreTableRowsAndColumns = ({
         if (sortBy) {
             if (sortOrder === 'asc') {
                 dataToSort.sort((a, b) => {
+                    if (a[sortBy] === true) return 1;
```

Comment:
Do we need to handle booleans as a whole so that false values dont attempt to get stringified? When i sort on a column that contains booleans and undefined, seems like undefined is getting handled in unexpected ways!

Reply:
woah nice!! I think the more robust will be much more maintainable and easier to logic thru all the branches. Thank you! I left one comment about a possible return type, otherwise it looks solid and behaves how Id expect